Scripts/Runtime/Entities/Tasks/TaskFlow/Nodes/JobNodeLookup.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+
+namespace Anvil.Unity.DOTS.Entities.Tasks
+{
+    internal class JobNodeLookup : AbstractNodeLookup
+    {
+        private static readonly TaskFlowRoute[] TASK_FLOW_ROUTE_VALUES = (TaskFlowRoute[])Enum.GetValues(typeof(TaskFlowRoute));
```

Comment:
I'm not sure I understand. I can't just put it in the `TaskFlowRoute` file and it can't be inside the enum. Where would you want to see this?

Reply:
Normally I'd have an extension or util class next to the enum that provided utility functionality.
Looking at your use of these again it doesn't match the usual case where you already have an enum value and want to get all values off of it....making the pattern a bit nicer.
```csharp
TaskFlowRoute someRoute;
foreach(TaskFlowRoute route in someRoute.AllValues()){...}
```

Since you just want the enum values from no existing value then that pattern doesn't make as much sense and a utility class with this array stored in it wouldn't be very discoverable so it's probably fine to just leave as is.

If we end up creating this array a lot we can solve the problem then.